### ISSUE TITLE
Stuck on Pending

### ISSUE THREAD

**Issue by Walkingwalkerwalks** [waylaidwanderer/FicSave#178]
It seems to have occurred again. I don't know how long it's been happening now after you resolved the issue 11 hours ago since I just used the site a minute ago, but it's the same problem as yesterday.

UPDATE: Some of its current issues can be chalked up to fanfiction.net being under maintenance that causes the download progress to go on and off at times. Once the site is more stable (Hopefully in a couple of hours or a day), I will check up on ficsave's stability again.

**Comment by Arw165** [comment]
I tried downloading from FFnet right after seeing the previous thread closed to test the site and downloading became practically impossible. It would be stuck on pending for some time, then it would start loading chapters "1 chapter per minute" (super slow) only to show 'error' and 'fail' at some point.

But before writing this comment I downloaded 7 fics from FFnet quite fast. So maybe everything is back to normal.
